core/frontend/src/menus.ts

```diff
@@ -16,6 +16,14 @@ const menus = [
         text: `General Autopilot settings. Allows you to start/stop Ardupilot (if using Navigator or SITL), or update
         the firmware of your flight controller.`,
       },
+      {
+        title: 'Pings',
+        icon: 'mdi-radar',
+        route: '/autopilot/pings',
+        advanced: false,
+        text: 'Ping devices discovery and management. This allows you to see all detected Ping devices connected to '
+              + 'either your Companion Computer or its local network',
```

Comment:
idk about this one. Companion computer is a concept that is not going anywhere anytime soon. But I do worry about confusing our users...

Reply:
I think the idea was that when there's a separate flight controller, it's a companion computer, but since Navigator can't operate without the computer then "Onboard Computer" is a better term for general use (since it encompasses both options).

Reply:
ok, I'm convinced.